source/common/dynamo/dynamo_filter.cc

```diff
@@ -35,6 +35,7 @@ Http::FilterDataStatus DynamoFilter::decodeData(Buffer::Instance& data, bool end
   if (end_stream) {
     return Http::FilterDataStatus::Continue;
   } else {
+    // Buffer until the complete request has been processed.
```

Comment:
Oh I'm also happy to remove all the boilerplate comments once you've taken a look, Matt, I just wanted a way to call all unchanged returns of StopIterationAndBuffer to your attention.  Let me know when/if you want them gone.